packages/vuetify/src/util/helpers.ts

```diff
@@ -214,9 +214,10 @@ export function deepEqual (a: any, b: any): boolean {
   return props.every(p => deepEqual(a[p], b[p]))
 }
 
-export function getObjectValueByPath (obj: object, path: string, fallback?: any): any {
+export function getObjectValueByPath (obj: any, path: string, fallback?: any): any {
   // credit: http://stackoverflow.com/questions/6491463/accessing-nested-javascript-objects-with-string-key#comment55278413_6491621
-  if (!path || path.constructor !== String) return fallback
+  if (obj == null || !path || path.constructor !== String) return fallback
```

Comment:
`typeof path`?

Reply:
:+1: https://stackoverflow.com/questions/39802506/constructor-vs-typeof-to-detect-type-in-javascript